README.md

```diff
@@ -1,3 +1,301 @@
+<!--
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
 # create-cloud-deploy-release
 
-This action creates a release for Cloud Deploy.
+The `create-cloud-deploy-release` GitHub Action creates a [Cloud Deploy][cd]
+[release][cd-release] to manage the deployment of an application to one or more
+[Google Kubernetes Engine (GKE)][gke] or [Cloud Run][cloud-run]
+[targets][cd-target].
+
+## Prerequisites
+
+-   This action requires Google Cloud credentials that are authorized to access
+    the secrets being requested. See [Authorization](#authorization) for more
+    information.
+
+-   This action runs using Node 16. If you are using self-hosted GitHub Actions
+    runners, you must use runner version
+    [2.285.0](https://github.com/actions/virtual-environments) or newer.
+
+-   This action depends on the existence of a [Cloud Deploy][cd]
+    [delivery pipeline][cd-pipeline] that is configured for the targets to which
+    the application will be deployed.
+
+## Usage
+
+```yaml
+jobs:
+  job_id:
+    # ...
+
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    steps:
+    - uses: 'actions/checkout@v3'
+
+    - uses: 'google-github-actions/auth@v1'
+      with:
+        workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+        service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
+
+   - id: create-release
+     uses: 'google-github-actions/create-cloud-deploy-release@v1'
+      with:
+        name: 'demo-app-${GITHUB_SHA::7}'
+        description: '${{ github.event.head_commit.message }}'
+        delivery_pipeline: '${{ env.PIPELINE }}'
```

Comment:
These env's aren't defined. For the quickstart here, I'd recommend hard-coding stuff.